src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final Random random = new Random();
```

Comment:
Avoid using static members for the `Random` instance. Instead, you can instantiate `Random` as a regular object in the constructor or directly in the method where it's used.